site/class-1-bar/sebasschmidt/readme.md

```diff
@@ -0,0 +1,11 @@
+# Hours of sleep 
```

Comment:
Your bar chart looks great (albeit late), but you readme could use some work, it’s missing some sections. Please see the [description of Readme again](https://github.com/cmda-fe3/course-17-18/blob/master/class-2.md#description)